pkg/subctl/cmd/verify.go

```diff
@@ -179,19 +207,14 @@ func clusterNameFromConfig(kubeConfigPath, kubeContext string) string {
 }
 
 func checkValidateArguments(args []string) error {
-	if len(args) != 2 {
-		return fmt.Errorf("two kubeconfigs must be specified")
+	if len(args) != 2 && len(kubeContexts) != 2 {
+		return fmt.Errorf("two kubecontexts must be specified")
 	}
-	if strings.Compare(args[0], args[1]) == 0 {
-		return fmt.Errorf("kubeconfig file <kubeConfig1> and <kubeConfig2> cannot be the same file")
-	}
-	same, err := compareFiles(args[0], args[1])
```

Comment:
This removal will cause a regression, was this intentionally removed (instead of adjusted to only check if files were sent as args)?

Reply:
It was intentionally adjusted to only check if files were provided as arguments...

Reply:
I’ve restored the comparison.